app/assets/styles/bulletin.css

```diff
@@ -97,7 +97,7 @@ hr.image {
 	color: black;
 }
 
-.buarticle-violet {
+.buarticle-blue {
 	color: white;
 }
```

Comment:
This class is named blue but the color is white. That's very strange. 

I don't want to hold up this PR any longer, but we should decide how to handle colors. I'm okay mixing BEM and atomic classes, however, I think the classes should match their names. Take a look at the Tachyons project if you want to see a functionall/atomic css library and how their color classes match.

Reply:
Blue refers to the background color.

I agree we need better names for the classes, and suggest we create a new issues to improve the HTML semantic and CSS.

Please close the PR.  I want to move to the next step, if possible.